### ISSUE TITLE
problem with brightness/color

### ISSUE THREAD

**Issue by krobipd** [sahilchaddha/homebridge-magichome-platform#6]
hello

i have problem with the brightness, when I start HomeBridge with the leds on (switch on before with the magic home app) the app will recognize it.

but then the problems start first of when I try to dimm the leds with HomeBridge the color will change to another (in my case from slightly yellow/white to purple) but the brithness is ok - so 50% in the HomeBridge app are 50 % in the magic home app.

but when I then change the color the brightness gets wrong - its stays in the HomeBridge app at - for example - 50 % but the magic home app tells me its 41%.

then the its impossible the get 100% brightness with HomeBridge because 100% will be 84% in the magic home app - and when I change it to 100% in the magic home app it will get brighter.

**Comment by krobipd** [comment]
thx for your answer

I tried to add purewhite false but there is no change at all.

I use this wlan - led controller - [link](https://www.amazon.co.uk/gp/product/B01AA6221S/ref=as_li_ss_tl?ie=UTF8&psc=1&linkCode=sl1&tag=stephradfo-21&linkId=3984090298019c91f2bcc884fe5b2473)

here is part of my config

```
"platform": "MagicHome-Platform",
                        "debug": false,
                        "lights": [{
                                        "name": "living Wand",
                                        "ip": "192.168.1.5",
                                        "purewhite": false,
                                        "setup": "RGBWW"
```

![img_0450](https://user-images.githubusercontent.com/27220902/49681262-1262a980-fa9f-11e8-945c-acf887268cab.png)
![img_0449](https://user-images.githubusercontent.com/27220902/49681263-1262a980-fa9f-11e8-9c32-b93995b8ca7e.jpeg)

**Comment by sahilchaddha** [comment]
Hey @krobipd ,

I have debugged the issue and was able to reproduce it.
I will be creating PR containing potential fix to your problem.

**Comment by sahilchaddha** [comment]
@krobipd I have released `v1.3.0`. This should fix your issue in regards to incorrect color/brightness. I would recommend uninstalling previous version & reinstalling the latest one.
